docs/docs/01-start-here/02-installation.md

```diff
@@ -83,26 +83,17 @@ to use it, but it's great. It's available through the VSCode Marketplace [here](
 
 ## Wing Console
 
-:::caution Wing Console doesn't support Linux yet
+The Wing Console is a web application that provides a developer-friendly interface for viewing, exploring, and interacting with your Wing applications running on the local cloud simulator.
 
-Click :thumbsup: on [this issue](https://github.com/winglang/wing/issues/723) to indicate that you
-are looking for Linux support.
+To install the Wing Console, follow these steps:
```

Comment:
Since the console is included with the CLI, I think it's okay to remove this list